xcode/build_phases/swiftlint.rb

```diff
@@ -0,0 +1,393 @@
+        swift_files = SwiftFileManager.new(touchin_excluded_files, '')
+
+        included_files = included_files.select { |file_name| not swift_files.is_excluded_file(file_name) }
+        included_files = included_files.map { |file_path| file_path.add_back_to_path(depth_git_count) }
+        
+        touchin_swiftlint_yaml_manager.update('excluded', [])
+        touchin_swiftlint_yaml_manager.update('included', included_files)
+        
+        if not included_files.nilOrEmpty?
+            make_single_strategy(@touchin_swiftlint_yaml_temporary_path)
+        else
+            puts 'Git found the swift files to check, but they are excluded in yaml'
+        end
+    end
+    
+    private
+    
+    def make_single_strategy(swiftlint_yaml_path)
+        result_swiftlint_command = get_swiftlint_command(swiftlint_yaml_path)
+        puts result_swiftlint_command
+        make_bash_command(result_swiftlint_command)
+    end
+    
+    def make_multiple_strategy(touchin_swiftlint_yaml_temporary_path, old_swiftlint_yaml_temporary_path)
+        touchin_swiftlint_command = get_swiftlint_command(touchin_swiftlint_yaml_temporary_path)
+        old_swiftlint_command = get_swiftlint_command(old_swiftlint_yaml_temporary_path)
+        result_swiftlint_command = touchin_swiftlint_command + ' && ' + old_swiftlint_command
+        puts result_swiftlint_command
+        make_bash_command(result_swiftlint_command)
+    end
+    
+    def get_swiftlint_command(swiftlint_yaml_path)
+        autocorrect_command = @swiftlint_autocorrect_command + swiftlint_yaml_path
+        lint_command = @swiftlint_lint_command + swiftlint_yaml_path
+        return autocorrect_command + ' && ' + lint_command
+    end
+    
+    def make_bash_command(bash_command)
+        exit (exec bash_command)
+    end
+    
+    def create_copy_temporary_files
+        create_copy_temporary_touchin_files
+        FileUtils.cp @old_swiftlint_yaml_path, @old_swiftlint_yaml_temporary_path
+    end
+    
+    def create_copy_temporary_touchin_files
+        Dir.mkdir(@temporary_swiftlint_folder_name) unless Dir.exist?(@temporary_swiftlint_folder_name)
+        FileUtils.cp @touchin_swiftlint_yaml_path, @touchin_swiftlint_yaml_temporary_path
+    end
+end
+
+setting = SettingOption.new
+maker = StrategyMaker.new(setting.source_directory, setting.pods_directory, setting.touchin_swiftlint_yaml_path)
+
+if setting.check_mode.eql? 'fully' and setting.use_multiple.true?
+    maker.make_fully_multiple_strategy(setting.source_date)
+elsif setting.check_mode.eql? 'fully' and not setting.use_multiple.true?
+    maker.make_fully_single_strategy
+elsif setting.check_mode.eql? 'simplified' and setting.use_multiple.true?
+    maker.make_simplified_multiple_strategy(setting.source_date, setting.depth_git_count)
+elsif setting.check_mode.eql? 'simplified' and not setting.use_multiple.true?
+    maker.make_simplified_single_strategy(setting.depth_git_count)
```

Comment:
make_ or run_?

Reply:
it seemed to me `maker` is `make`

Reply:
make -> should return something (shell command, for example)

Reply:
make -> not necessarily should return something, for me
where have you seen this?
i can to change to `run_`, okey.